### PR TITLE
migrate dummy content

### PR DESCRIPTION
**Pull request by amitaibu**
Content types to migrate:
- Repo (group)

CI:
- CI build
- CI incident (skip because they are creating automatically #29)

UI:
- UI build
- Screenshot

### CODE REVIEW

File: shoov/modules/custom/shoov_restful/plugins/restful/restful_token_auth/token_auth/access_token/1.0/ShoovGithubAuthAuthentication.class.php [hunk 1]
```diff
@@ -42,7 +42,11 @@ protected function getUser() {
         'client_id' => variable_get('shoov_github_client_id'),
         'client_secret' => variable_get('shoov_github_client_secret'),
         'code' => $request['code'],
+        'X-NewRelic-ID' => 'XQUCV15VGwYJXVhTBAg=',
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_restful/plugins/restful/restful_token_auth/token_auth/access_token/1.0/ShoovGithubAuthAuthentication.class.php]: this should be removed, it was a debug

**Comment by solker** [comment]
0e09b9bc042e462af2490d4c73ed33b345ec595f Fix Users, Repositories, Ci Builds migrations.
selection_036

**Comment by solker** [comment]
390af10 Make UI Build migrate without Screenshots.
selection_037
selection_038

**Comment by solker** [comment]
f06961a Finish ShoovScreenshotsMigrate migration.
selection_040
selection_039
selection_041

**Comment by amitaibu** [comment]
Lets use different images for the baseline/diff/regression

**Comment by solker** [comment]
2d0285e

Users:
selection_043

Screenshot:
selection_044

I am using same images for screenshots because don't have another.
selection_045

**Comment by solker** [comment]
@amitaibu ready for code review.

Notice:
A content type Screenshot contain unfamiliar label for the field_build.
selection_046

File: shoov/modules/custom/shoov_migrate/handlers/node/ShoovScreenshotsMigrate.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+  }
+
+  /**
+   * Assign this screenshot with Ui Build.
+   *
+   * @param $entity
+   * @param $row
+   */
+  public function complete($entity, $row) {
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_migrate/handlers/node/ShoovScreenshotsMigrate.php:86]: remove - this is wrong.

File: shoov/modules/custom/shoov_migrate/handlers/user/ShoovUsersMigrate.php [hunk 3]
```diff
@@ -64,4 +60,14 @@ public function __construct() {
...
   }
+
+  /**
+   * Overrides Migration::prepareRow().
+   *
+   * Add default email and randomly generated Github token.
+   */
+  public function prepareRow($row) {
+    $row->_email = strtolower($row->_username) . '@example.com';
+    $row->_github_token = sha1(srand());
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_migrate/handlers/user/ShoovUsersMigrate.php]: `sha1(srand());` => `drupal_random_key()`

File: shoov/modules/custom/shoov_restful/plugins/restful/restful_token_auth/token_auth/access_token/1.0/ShoovGithubAuthAuthentication.class.php [hunk 4]
```diff
@@ -41,8 +41,10 @@ protected function getUser() {
       'data' => http_build_query(array(
         'client_id' => variable_get('shoov_github_client_id'),
         'client_secret' => variable_get('shoov_github_client_secret'),
-        'code' => $request['code'],
+        'code' => $request['code']
       )),
+      'headers' => array(
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_restful/plugins/restful/restful_token_auth/token_auth/access_token/1.0/ShoovGithubAuthAuthentication.class.php]: remove.

File: shoov/modules/custom/shoov_restful/plugins/restful/restful_token_auth/token_auth/access_token/1.0/ShoovGithubAuthAuthentication.class.php [hunk 5]
```diff
@@ -41,8 +41,10 @@ protected function getUser() {
       'data' => http_build_query(array(
         'client_id' => variable_get('shoov_github_client_id'),
         'client_secret' => variable_get('shoov_github_client_secret'),
-        'code' => $request['code'],
+        'code' => $request['code']
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_restful/plugins/restful/restful_token_auth/token_auth/access_token/1.0/ShoovGithubAuthAuthentication.class.php]: keep the trailing comma

File: shoov/modules/custom/shoov_migrate/handlers/node/ShoovScreenshotsMigrate.php (hunk 2 above)
Comment (solker) [inline shoov/modules/custom/shoov_migrate/handlers/node/ShoovScreenshotsMigrate.php:86]: @amitaibu 
Screenshot have ui_build field and it's required.
selection_048

UI Build have PR Screenshot IDs field and it's not required. 
selection_047

By this complete function i fill PR screenshots IDs in exists UI Build. 
This field is using somewhere? Are you sure that we don't need fill it?

**Comment by solker** [comment]
e627f89
selection_049
selection_050

**Comment by amitaibu** [comment]
Can you please change the yndex image to another English one :)

**Comment by solker** [comment]
26a7cbd
selection_051

File: shoov/modules/custom/shoov_restful/plugins/restful/restful_token_auth/token_auth/access_token/1.0/ShoovGithubAuthAuthentication.class.php [hunk 6]
```diff
@@ -42,7 +42,7 @@ protected function getUser() {
         'client_id' => variable_get('shoov_github_client_id'),
         'client_secret' => variable_get('shoov_github_client_secret'),
         'code' => $request['code'],
-      )),
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_restful/plugins/restful/restful_token_auth/token_auth/access_token/1.0/ShoovGithubAuthAuthentication.class.php:45]: minor: always keep the trailing comma

**Comment by solker** [comment]
c025703
I have added coma.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**